package.json

```diff
@@ -125,8 +125,8 @@
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "start:docs": "yarn build:tokens && lerna run develop --scope=@patternfly/react-docs --stream",
     "storybook": "yarn storybook:run",
-    "storybook:run": "start-storybook -c storybook -p 6006",
-    "storybook:build": "build-storybook -c storybook -o .out",
+    "storybook:run": "yarn build && start-storybook -c storybook -p 6006",
+    "storybook:build": "yarn build && build-storybook -c storybook -o .out",
```

Comment:
Is this intended as part of this PR?

Reply:
I can separate it out if needed, but yes. Seems people are still having issues with running storybook and not building first.